src/pages/Dashboard/sub-pages/Vaults/VaultsTable/index.tsx

```diff
@@ -229,111 +201,51 @@ const VaultsTable = (): JSX.Element => {
     [t]
   );
 
-  const [filteredVaults, setFilteredVaults] = React.useState<Array<VaultExt>>([]);
-  const handleError = useErrorHandler();
-  React.useEffect(() => {
-    if (!vaultsExt) return;
-    if (!handleError) return;
-
-    (async () => {
-      try {
-        const theFilteredVaults = await asyncFilter<VaultExt>(vaultsExt, async (vaultExt: VaultExt) =>
-          VAULT_COLLATERAL_TOKENS.some(async (item) => {
-            const collateralToken = await currencyIdToMonetaryCurrency(
-              window.bridge.assetRegistry,
-              vaultExt.id.currencies.collateral
-            );
-            const isEqual = isCurrencyEqual(item, collateralToken);
-
-            return isEqual;
-          })
-        );
-
-        setFilteredVaults(theFilteredVaults);
-      } catch (error) {
-        handleError(error);
-      }
-    })();
-  }, [vaultsExt, handleError]);
```

Comment:
This filtering is no longer needed since the collateral currencies will be fetched from chain.